src/components/Buttons.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/require-default-props */
+/* eslint-disable react/default-props-match-prop-types */
+import React from 'react';
+import Button from './button';
+
```

Comment:
hello bro, please rename then this file as `BottonPanel.js`, to not confuse with the other file. Hope you understand this

Reply:
ok, i understand that points and i already did hat changes. Thanks for the review